tests/tools/test_storage.py

```diff
@@ -238,20 +217,21 @@ async def test_list_directories_by_name(self, setup_test_directory):
         ]
         test_path, _ = setup_test_directory(dir_specs)
 
-        result = await storage.list_directories(str(test_path), order_by="name")
+        result = await mcp_client.call_tool("list_directories", arguments={"path": str(test_path), "order_by": "name"})
+        result_text = result.content[0].text
 
-        assert isinstance(result, str)
         # Verify all directories are present
-        assert "alpha" in result
-        assert "beta" in result
-        assert "gamma" in result
+        assert "alpha" in result_text
+        assert "beta" in result_text
+        assert "gamma" in result_text
+
         # Verify sorted order (alpha should appear before beta, beta before gamma)
-        alpha_pos = result.find("alpha")
-        beta_pos = result.find("beta")
-        gamma_pos = result.find("gamma")
+        alpha_pos = result_text.find("alpha")
+        beta_pos = result_text.find("beta")
+        gamma_pos = result_text.find("gamma")
```

Comment:
I guess this is not needed given that we search the text position below. Maybe we could just cut this part and rely on the `result_text.find()`

```python
        # means the text was not found in the `result_text` variableV
        assert all(pos > -1 for pos in [alpha_pos, beta_pos, gamma_pos])

```